test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -369,6 +376,12 @@ namespace integration_framework {
     /// Get the controlled Iroha instance.
     const std::shared_ptr<IrohaInstance> &getIrohaInstance() const;
 
+    /// Set the ITF peer keypair and initialize irohad pipeline.
+    void initPipeline(const shared_model::crypto::Keypair &keypair);
+
+    /// Start the ITF.
+    void subscribeQueuesAndRun();
```

Comment:
Does it seem that for each ITF test we will invoke this method? I suggest preserving old behavior as it is and to add additional API for fake peer case.

Reply:
You could notice that CI fails, but if you look which tests fail, you see that it is only the fake peer example test, which I explained in a comment above. You could also notice that I did not do modifications to all present ITF usage. From that you cold conclude that the old interface still works as it is.

What really changed in the interface is the ways one can start the ITF. And these changes seem inevitable for the reasons stated in the PR description.